node/clients/pricingClient.ts

```diff
@@ -18,8 +18,21 @@ export default class PricingClient extends ExternalClient {
     })
   }
 
-  public getPrice(skuId: string | number, priceTable: string | undefined) {
-    return this.http.get<Quote>(`/prices/${skuId}/computed/${priceTable}`)
+  public async getPrice(
+    skuId: string | number,
+    priceTable: string | undefined
+  ): Promise<Quote | undefined> {
+    try {
+      return await this.http.get<Quote>(
+        `/prices/${skuId}/computed/${priceTable}`
```

Comment:
I'm not sure if can exist a priceTable called `undefined`. But, if this is possible, it will work like a default value (if no priceTable is set, the priceTable === undefined will be returned. Is it the expected behavior? If it's not, you can check priceTable value before the HTTP call.

Reply:
Good point! Fixed!